portfolio/src/main/webapp/index.html

```diff
@@ -9,8 +9,39 @@
   <body>
     <div id="content">
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
```

Comment:
Looks like this message is part of the random greeting button below. Instead of deleting this, should we move it down there?